app/components/validate-appointment.js

```diff
@@ -7,21 +7,68 @@ export default Ember.Component.extend({
     Ember.run.scheduleOnce('afterRender', this, function(){
       validateInputs();
       validateForm();
+      validateRequestForm();
+      validateRequestInputs();
       validateCheckCheckBoxInput();
     });
 
+    function validateRequestInputs(){
+      Ember.$('#people-count').focusout(function(){
+        return checkInput(this);
+      });
+
+      Ember.$('#people-count').focus(function(){
```

Comment:
Can we do something like this?
```
["#people-count", "#description", ".district-selector select"].forEach(selector => {
  Ember.$(selector).focus(function(){
    return removeHighlight(this);
  });
})
```

And possibly for other events also?

Reply:
sure @Darshan-Chauhan, i'll try this, and also i'm thinking of moving request purpose validation to a seperate component.
thoughts ?

Reply:
Sounds good 👍